app/Http/Controllers/BiosController.php

```diff
@@ -62,24 +55,18 @@ public function edit($id)
         ]);
     }
 
-    public function update($id, Request $request)
+    public function update(UpdateBioRequest $request, $id)
```

Comment:
Since the `UpdateBioRequest` is the same as the `SaveBioRequest` can you use `SaveBioRequest` for both `store` and `update`?